pkg/visor/init.go

```diff
@@ -344,6 +345,59 @@ func initDmsgCtrl(ctx context.Context, v *Visor, _ *logging.Logger) error {
 	return nil
 }
 
+func initDmsgHTTPLogServer(ctx context.Context, v *Visor, log *logging.Logger) error {
+	dmsgC := v.dmsgC
+	if dmsgC == nil {
+		return nil
+	}
+	var dmsgTimeout = v.conf.LogRotationInterval
+	logger := v.MasterLogger().PackageLogger("dmsghttp_logserver").WithField("timeout", dmsgTimeout)
+
+	lis, err := dmsgC.Listen(uint16(uint(80)))
```

Comment:
Don't we typically store these ports somewhere inside the Skyenv or some similar place? I would not bury the static port in here like that.

Reply:
done